scipy/stats/unuran/unuran_wrapper.pyx.templ

```diff
@@ -761,9 +761,14 @@ cdef class TransformedDensityRejection(Method):
         >>> from scipy.stats import norm
         >>> from math import pi, sqrt, exp
         >>> 
-        >>> pdf = lambda x: 1./sqrt(2*pi) * exp(-0.5 * x**2)
-        >>> dpdf = lambda x: -x/sqrt(2*pi) * exp(-0.5 * x**2)
-        >>> rng = TransformedDensityRejection(pdf, dpdf, seed=123)
+        >>> class MyDist:
```

Comment:
should we drop the normalizing constant in the example and add a comment that it is not needed?

Reply:
Done.